Extensions/MaskingExtension.json

```diff
@@ -0,0 +1,113 @@
+{
```

Comment:
The name of the file must be the name of the extension. I suggest `SpriteMasking.json`